ui/compontents.go

```diff
@@ -336,6 +342,212 @@ func endToEndRow(gtx layout.Context, leftWidget, rightWidget func(C) D) layout.D
+			return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+				layout.Rigid(func(gtx C) D {
+					txt := page.theme.Body2(title)
+					txt.Color = page.theme.Color.Text
+					inset := layout.Inset{
+						Bottom: values.MarginPadding15,
+					}
+					return inset.Layout(gtx, txt.Layout)
+				}),
+				layout.Rigid(body),
+			)
+		})
+	}
+	wallAcctSelector := page.wallAcctSelector
+	w := []func(gtx C) D{
+		func(gtx C) D {
+			tit := page.theme.Label(values.TextSize24, wallAcctSelector.title)
+			tit.Color = page.theme.Color.Text
+			return tit.Layout(gtx)
+		},
+		func(gtx C) D {
+			return wallAcctSelector.wallets.Layout(gtx, len(page.info.Wallets), func(gtx C, windex int) D {
+				return wallAcctGroup(gtx, page.info.Wallets[windex].Name, func(gtx C) D {
+					return wallAcctSelector.accounts.Layout(gtx, len(page.info.Wallets[windex].Accounts), func(gtx C, aindex int) D {
+						click := wallAcctSelector.walletAccounts[windex][aindex].evt
+						pointer.Rect(image.Rectangle{Max: gtx.Constraints.Max}).Add(gtx.Ops)
+						click.Add(gtx.Ops)
+						page.walletAccountsHandler(gtx, wallAcctSelector.walletAccounts[windex][aindex])
+						return page.walletAccountLayout(gtx, wallAcctSelector.walletAccounts[windex][aindex])
+					})
+				})
+			})
+		},
+	}
+
+	return wallAcctSelector.walletAccount.Layout(gtx, w, 850)
+}
+
+func (page *pageCommon) walletAccountsHandler(gtx layout.Context, wallAcct walletAccount) {
+	for _, e := range wallAcct.evt.Events(gtx) {
+		if e.Type == gesture.TypeClick {
+			*page.selectedWallet = wallAcct.walletIndex
+			*page.selectedAccount = wallAcct.accountIndex
+			page.wallAcctSelector.isWalletAccountModalOpen = false
+		}
+	}
+}
+
+func (page *pageCommon) walletAccountLayout(gtx layout.Context, wallAcct walletAccount) layout.Dimensions {
+	accountIcon := page.icons.accountIcon
+	accountIcon.Scale = 1
+	inset := layout.Inset{
+		Bottom: values.MarginPadding10,
+	}
+	return inset.Layout(gtx, func(gtx C) D {
+		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				return layout.Flex{}.Layout(gtx,
+					layout.Flexed(0.1, func(gtx C) D {
+						inset := layout.Inset{
+							Right: values.MarginPadding10,
+							Top:   values.MarginPadding15,
+						}
+						return inset.Layout(gtx, func(gtx C) D {
+							return accountIcon.Layout(gtx)
+						})
+					}),
+					layout.Flexed(0.8, func(gtx C) D {
+						return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+							layout.Rigid(func(gtx C) D {
+								accountLabel := page.theme.Body2(wallAcct.accountName)
+								accountLabel.Color = page.theme.Color.Text
+								accountBalLabel := page.theme.Body2(wallAcct.totalBalance)
+								accountBalLabel.Color = page.theme.Color.Text
+								return page.accountTableLayout(gtx, accountLabel, accountBalLabel)
+							}),
+							layout.Rigid(func(gtx C) D {
+								spendibleLabel := page.theme.Body2("Spendable")
+								spendibleLabel.Color = page.theme.Color.Gray
```

Comment:
Kindly correct the spelling here. it should be "spendable", not "spendible". Also try not to add the data type to the variable name, unless it contradicts with its use on the page. For instance, using if "accounts" as a variable name of a List layout would lead to confusion in the code, it would be perfectly fine to make the variable name "accountsList". However, spendable seems just fine here.